prodmgr/parser.py

```diff
@@ -99,5 +99,20 @@ def create_parser():
         help='The hostname of the container image registry',
         default=DEFAULT_CONTAINER_REGISTRY_HOSTNAME,
     )
+    parser.add_argument(
+        '--deletion-container-name',
+        help='The full path and file name of the deletion container',
+        default='artifactory.algol60.net/csm-docker/stable/product-deletion-utility',
+    )
+    parser.add_argument(
+        '--deletion-container-version',
+        help='The version of the deletion container',
+        default='0.0.0-product-deletion-utility',
```

Comment:
Is this the convention we're using for version tags on `product-deletion-utility` images? IMO it seems redundant with image name.

Reply:
Good observation. No, it is not the convention that I'm taking. Instead, as part of CMS' cms-meta-tools, it reaches out to determine what the latest version of the product-deletion-utility container is. It then looks for the string '0.0.0-product-deletion-utility' and substitutes that version for this string. Thus, it isn't a convention for tagging the image. It's just an internal convention to ensure the right string is substituted.